.eslintrc.js

```diff
@@ -15,4 +18,46 @@ module.exports = {
   'plugins': [
     'react',
   ],
+  'overrides': [
+    {
+      'files': ['src/client/**/*.js'],
+      'parserOptions': {
+        'sourceType': 'module',
+      },
+      'env': {
+        'browser': true,
+        'node': true,
```

Comment:
do you really need the node environment enabled in the front end code?

Reply:
we need to use 'require', it seems to be node grammar.